apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -153,8 +157,20 @@ export default function LearningGoal({
         <div className={style.learningGoalHeaderRightSide}>
           {aiEnabled && <AiToken />}
           {/*TODO: Display status of feedback*/}
-          {canProvideFeedback && (
-            <BodyThreeText>{i18n.needsApproval()}</BodyThreeText>
+          {canProvideFeedback &&
+            aiEnabled &&
+            understandingLevel.current === -1 && (
```

Comment:
I think this should use `displayUnderstanding` instead, right? Maybe asked a different way: why use the ref over the state here?

Reply:
I removed the understandingLevel ref completely, and it seems to be working fine. Had included that because I was worried about the value not updating until re-render, but it seems to be ok.